genie-proto/build.gradle

```diff
@@ -0,0 +1,81 @@
+apply plugin: 'com.google.protobuf'
+
+ext {
+    protogenIgnorePattern = "**/com/netflix/genie/protogen/*"
+}
+
+dependencies {
+
+    /*******************************
+     * Compile Dependencies
+     *******************************/
+
+     compile ("com.google.protobuf:protobuf-java")
+     compile("io.grpc:grpc-stub")
+     compile("io.grpc:grpc-protobuf")
+
+    /*******************************
+     * Provided Dependencies
+     *******************************/
+
+    /*******************************
+     * Runtime Dependencies
+     *******************************/
+
+    /*******************************
+     * Test Dependencies
+     *******************************/
+
+}
+
+// Configure protobuf classes and grpc stubs generation
+protobuf {
+    protoc {
+        // Download the protoc compiler
+        artifact = 'com.google.protobuf:protoc'
+        // Use project convention for generated files
+        generatedFilesBaseDir = file("${buildDir}/generated-grpc")
```

Comment:
Why not reference the same variable from the root build file?

Reply:
The value is not the same, but let me see if I can refactor this to avoid hardcoding the same path twice